src/pages/EnablePage.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line
+import {BrowserRouter as Router, Link, Route, Routes} from 'react-router-dom';
```

Comment:
It might be a good idea to have template css files, which can be used when creating new pages